hashMapFile.java

```diff
@@ -0,0 +1,45 @@
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+    import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+    public class hashMapFile {

```

Comment:
```suggestion
    public class HashMapFile {
```
class name should start with uppercase

Reply:
fixed in 766320d 'changes to task3'